BREAKING.md

```diff
@@ -17,35 +17,6 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 - Avoid using code formatting in the title (it's fine to use in the body).
 - To explain the benefit of your change, use the [What's New](https://fluidframework.com/docs/updates/v1.0.0/) section on FluidFramework.com.
 
-# 3.0.0
-
-## 3.0.0 Upcoming changes
-- [Remove `type` field from `ShareLinkInfoType`](#Remove-type-field-from-ShareLinkInfoType)
-- [Remove `ShareLinkTypes` interface](#Remove-ShareLinkTypes-interface)
-- [Remove `enableShareLinkWithCreate` from `HostStoragePolicy`](#Remove-enableShareLinkWithCreate-from-HostStoragePolicy)
-- [Add assertion that prevents sending op while processing another op](#add-assertion-that-prevents-sending-op-while-processing-another-op)
--
-### Remove `type` field from `ShareLinkInfoType`
-This field has been deprecated and will be removed in a future breaking change. You should be able to get the kind of sharing link from `shareLinkInfo.createLink.link` property bag.
-
-### Remove `ShareLinkTypes` interface
-`ShareLinkTypes` interface has been deprecated and will be removed in a future breaking change. Singnature of `createOdspCreateContainerRequest` has been updated to now accept `ISharingLinkKind` property instead.
-```diff
-    function createOdspCreateContainerRequest(
-        siteUrl: string,
-        driveId: string,
-        filePath: string,
-        fileName: string,
--       createShareLinkType?: ShareLinkTypes,
-+       createShareLinkType?: ShareLinkTypes | ISharingLinkKind,
-    ):
-```
-
-
-### Remove `enableShareLinkWithCreate` from `HostStoragePolicy`
-`enableShareLinkWithCreate` feature gate has been deprecated and will be removed in a future breaking change. If you wish to enable creation of a sharing link along with the creation of Fluid file, you will need to provide `createShareLinkType:ISharingLinkKind` input to the `createOdspCreateContainerRequest` function and enable the feature using `enableSingleRequestForShareLinkWithCreate` in `HostStoragePolicy`
-### Add assertion that prevents sending op while processing another op
-`preventConcurrentOpSend` has been added and enabled by default. This will run an assertion that closes the container if attempting to send an op while processing another op. This is meant to prevent non-deterministic outcomes due to concurrent op processing.
 # 2.0.0
 
 ## 2.0.0 Upcoming changes
```

Comment:
Is the intention here to advertise 2.0.0-internal-1.2.0 or 2.0.0-internal-2.0.0 ? I assume we can advertise "actionable" items in the minor release (as described here: https://github.com/microsoft/FluidFramework/wiki/Communicating-breaking-changes#breaking-changes-vs-upcoming-changes)

Reply:
The intention should be to advertise changes that are coming in a future major release that are currently actionable. So it could be `2.0.0-internal-2.0.0`, but it could also be a major release after that

Reply:
I do not think it can be any/arbitrary release? If the changes is "actionable" it means someone made some change in either main or next and that change can be acted upon. Dev can either add changes in the "main" in which case changes immediately go under "Breaking Changes" or they have added changes in "next" and here they go under "2.0.0-internal-2.0.0 breaking changes"

Reply:
Disregard my previous comment, it's incorrect. Take a look at https://github.com/microsoft/FluidFramework/pull/12161. I think we should be aligned here.